packages/pg/collection.js

```diff
@@ -56,6 +56,32 @@ const QBProto = Meteor.isClient ?
   Knex.QueryBuilder.prototype :
   Knex.Client.prototype.QueryBuilder.prototype;
 
+// Addresses https://github.com/meteor/postgres-packages/issues/9
+QBProto.fetchOne = function fetchOne() {
+  return QBProto.fetch.call(this)[0];
+};
+
+// Addresses https://github.com/meteor/postgres-packages/issues/9
+QBProto.fetchValue = function fetchValue(column) {
+  const row = QBProto.fetch.call(this)[0];
+  if ('undefined' === typeof column) {
```

Comment:
Hmm. I prefer the non-dependency on a library, but if that's best practice I'll follow suit.